testing/src/test/java/org/motechproject/nms/testing/it/api/MobileAcademyControllerBundleIT.java

```diff
@@ -147,7 +141,7 @@ public void testGetCourseValid() throws IOException, InterruptedException {
         HttpGet request = RequestBuilder.createGetRequest(endpoint);
 
         HttpResponse httpResponse = SimpleHttpClient.httpRequestAndResponse(request, RequestBuilder.ADMIN_USERNAME, RequestBuilder.ADMIN_PASSWORD);
-        assertTrue(httpResponse.getStatusLine().getStatusCode() == 200);
+        assertEquals(200, httpResponse.getStatusLine().getStatusCode());
```

Comment:
not a big fan of this style personally. I like to assert on the boolean of my expected assumption. oh well.